kubeflow/tf-serving/tf-serving.libsonnet

```diff
@@ -1,4 +1,6 @@
 {
+  util:: import "kubeflow/tf-serving/util.libsonnet",
```

Comment:
Can you explain the rationale for this tf-serving util as opposed to the core one?
```
local util = import "kubeflow/core/util.libsonnet"
```

Reply:
That's a different package. Users may use tf-serving package without core.